storage/internal/benchmarks/README.md

```diff
@@ -24,6 +24,9 @@ This runs 1000 iterations on 512kib to 2Gib files in the background, sending out
 | -gc_f | whether to force garbage collection <br> before every write or read benchmark |  `true` or `false` (present/not present) | `false` |
 | -min_size | minimum object size in bytes | any positive integer | `512` |
 | -max_size | maximum object size in bytes | any positive integer | `2 097 152` (2 GiB) |
+| -range_size | size of the range to read in bytes | any positive integer | `0` (<=0 reads the full object) |
```

Comment:
Wondering if we should have a max/min range size rather than a fixed size in order to allow a mix of sizes, as we do for full objects. What do you think?

Reply:
Perhaps, but right now the workload we are looking at uses specific sizes. I'm wary of adding more parameters that won't be used. Is there a specific experiment you'd like to see with varying sizes?